swap/int256/uint256.go

```diff
@@ -109,7 +122,7 @@ func (u *Uint256) String() string {
 
 // MarshalJSON implements the json.Marshaler interface
 // it specifies how to marshal a Uint256 struct so that it can be written to disk
-func (u Uint256) MarshalJSON() ([]byte, error) {
+func (u *Uint256) MarshalJSON() ([]byte, error) {
```

Comment:
The same question as for Int256.MarshalJSON.

Reply:
deal with as well